git.tex

```diff
@@ -1,11 +1,38 @@
-\subsection{Development Model}
+\subsection{Development Model}\label{sec:development}
 
-Lots of small repositories.
+To enable concurrent development across all six LSST sites, the LSST development team uses a decentralized development model based on the Git version control system\footnote{\url{https://git-scm.com/}}.
+[TODO: describe in terms of Agile values?]
 
-Feature branches associated with Jira tickets.
+\subsubsection{Version Control}\label{sec:git}\label{sec:subversion}
 
-Branch protection.
+[TODO: why did LSST switch from Subversion to Git?]
 
-Rebase before merging.
+\subsubsection{Code Organization}\label{sec:git_repositories}
+
+The data management system is divided into several hundred individual Git repositories, hosted on GitHub\footnote{\url{http://github.com/}}.
```

Comment:
Point to the https://github.com/lsst location.

Reply:
Per your comment on `#dm`, should this be a full reference? It seems a bit weird.

Reply:
I don't think so. It's not really something citable. It's a link to the LSST repositories, not an article or guide.